src/main/antlr/bio/terra/grammar/SQL.g4

```diff
@@ -68,6 +68,7 @@ expr : number
 
 column_expr : dataset_name '.' table_name '.' column_name
     | alias_name '.' column_name
+    | column_name
```

Comment:
Why does this need to change?

Reply:
When I included the full column name in the filter (e.g. `WHERE dataset_name.table_name.column_name = value`), the ANTLR validation passes, but the bigquery query fails. Including just the column name works though and keeps the filter string simpler.